# message type that corresponds to inchworm.py
Header header
int32[] inchworm_count
int32[] ticks
int32[] state_counts